test/viewport.test.js

```diff
@@ -54,4 +84,19 @@ describe('o-viewport', function() {
 	it('should get the orientation of the viewport', function() {
 		expect(oViewport.getOrientation() === 'portrait' || oViewport.getOrientation() === 'landscape').to.be(true);
 	});
+
+	it('should stop listening to scroll event', function(done) {
+		oViewport.stopListeningTo('scroll');
+		document.body.addEventListener('oViewport.scroll', function(ev) {
+			expect(false).to.be.true;
```

Comment:
![quote-william-s -burroughs-nothing-is-true-everything-is-permitted-124339_1](https://cloud.githubusercontent.com/assets/85783/6806103/aca69098-d23e-11e4-8c7e-81a4c5560005.png)